### PR TITLE
Forms Automated Test - LPS-133014 | master

### PR DESCRIPTION
**Pull request by Jamersonze**

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro [hunk 1]
```diff
@@ -117,6 +117,27 @@ definition {
 			locator1 = "FormFields#COLOR_SWATCH_ITEM");
 	}
 
+	macro editConfirmation {
```
Comment (carolmariaabb) [inline portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro:120]: @Jamersonze can you please rename this macro to `editRequireConfirmationField`? Thanks!

Comment (Jamersonze) [reply portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro]: Renamed

**Comment by Jamersonze** [comment]
Fixed

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**